diff --git a/Makefile b/Makefile
index 89fad5f..82735ce 100644
--- a/Makefile
+++ b/Makefile
@@ -145,54 +145,12 @@ installer-test:
 # ====================================================================
 # Dependencies
 
-# We list remotes so that if projects move hosts, we can pull from their new
-# canonical location.
-REMOTE_capnproto=https://github.com/sandstorm-io/capnproto.git
-REMOTE_ekam=https://github.com/sandstorm-io/ekam.git
-REMOTE_libseccomp=https://github.com/seccomp/libseccomp
-REMOTE_libsodium=https://github.com/jedisct1/libsodium.git
-REMOTE_node-capnp=https://github.com/kentonv/node-capnp.git
-
 deps: tmp/.deps
 
-tmp/.deps: deps/capnproto deps/ekam deps/libseccomp deps/libsodium deps/node-capnp
+tmp/.deps: deps/capnproto deps/ekam deps/libseccomp deps/libsodium
 	@mkdir -p tmp
-	@touch tmp/.deps
-
-deps/capnproto:
-	@$(call color,downloading capnproto)
-	@mkdir -p deps
-	git clone $(REMOTE_capnproto) deps/capnproto
-
-deps/ekam:
-	@$(call color,downloading ekam)
-	@mkdir -p deps
-	git clone $(REMOTE_ekam) deps/ekam
 	@ln -s .. deps/ekam/deps
-
-deps/libseccomp:
-	@$(call color,downloading libseccomp)
-	@mkdir -p deps
-	git clone $(REMOTE_libseccomp) deps/libseccomp
-
-deps/libsodium:
-	@$(call color,downloading libsodium)
-	@mkdir -p deps
-	git clone $(REMOTE_libsodium) deps/libsodium
-	@cd deps/libsodium && git checkout stable
-
-deps/node-capnp:
-	@$(call color,downloading node-capnp)
-	@mkdir -p deps
-	git clone $(REMOTE_node-capnp) deps/node-capnp
-
-update-deps:
-	@$(call color,updating all dependencies)
-	@$(foreach DEP,capnproto ekam libseccomp libsodium node-capnp, \
-	    cd deps/$(DEP) && \
-	    echo "pulling $(DEP)..." && \
-	    git pull $(REMOTE_$(DEP)) `git symbolic-ref --short HEAD` && \
-	    cd ../..;)
+	@touch tmp/.deps
 
 # ====================================================================
 # Ekam bootstrap and C++ binaries
